openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -178,6 +180,32 @@ const Carousel = {
                 }
             });
         }
+
+        function accessibilitySetup() {
+
+            function setA11yAttributes() {
+                //Ensuring offscreen elements do not receive focus
+                $(`${selector} [aria-hidden="true"] a[href]`).attr('tabindex', -1);
```

Comment:
Cache the result of the selector query before applying other queries to save DOM lookups.